src/component/Button.js

```diff
@@ -17,7 +17,7 @@ export default class Button extends React.Component {
   render() {
     const className = [
       "component-button",
-      this.props.orange ? "orange" : "",
+      this.props.orange ? "orange" : {},
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The ternary operator should return a string instead of an object when `this.props.orange` is true. This is because the className array is expected to contain strings representing CSS class names.
```diff
- this.props.orange ? "orange" : {},
+ this.props.orange ? "orange" : "",
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->